cloud/terraform/machineactuator.go

```diff
@@ -461,12 +572,15 @@ func (tf *TerraformClient) updateAnnotations(machine *clusterv1.Machine, masterE
 		machine.ObjectMeta.Annotations = make(map[string]string)
 	}
 	machine.ObjectMeta.Annotations[MasterIpAnnotationKey] = masterEndpointIp
+	machine.ObjectMeta.Annotations[ControlPlaneVersionAnnotationKey] = machine.Spec.Versions.ControlPlane
+	machine.ObjectMeta.Annotations[KubeletVersionAnnotationKey] = machine.Spec.Versions.Kubelet
+
 	_, err := tf.machineClient.Update(machine)
 	if err != nil {
 		return err
 	}
-	err = tf.updateInstanceStatus(machine)
-	return err
+	//err = tf.updateInstanceStatus(machine)
```

Comment:
What's wrong with this?

Reply:
I don't need it. It's not used anywhere. Instead I'm storing the versions I care about as annotations.